juniper_official/interface/check-ifl-state.rule

```diff
@@ -0,0 +1,144 @@
+/*
+ * Monitors sub interfaces state and notifies when anomalies are found.
+ * Two inputs control detection
+ *
+ *   1) ifd-name, is a regular expression that matches the interfaces
+ *      that you would like to monitor.  By default it's '.*', which matches
+ *      all interfaces. Use something like 'ge.*' to match only gigabit
+ *      ethernet interfaces. 
+ *
+ *   2) ifl-no, is a regular expression that matches the IFL index number that
+ *      you would like to monitor.  By default it's '.*', which matches all
+ *      interfaces. Use something like '0-10*' to match only 0 to 10 IFLs.
+ *
+ */
```

Comment:
Please add the requirement number for tracking

Reply:
Added